### PR TITLE
Improvements to the doc 

### PR DESCRIPTION
**Pull request by nimraahmed**
Changes made:
- Removed underline from next/prev buttons in the docs 
- added admonitions for info, tips, and notes
- added a section for developer guide with a coming soon tag
- added overview for the backend in contributor guide
- added doc cards for the local setup (working on customizing the icons)

### CODE REVIEW

File: docs/docs/contributor/server/basics/overview.mdx
Comment (khakimov) [inline docs/docs/contributor/server/basics/overview.mdx:1]: I think switching to TypeORM is still in progress.

**Review by khakimov — COMMENTED** [review]
👍

File: docs/src/css/custom.css [hunk 1]
```diff
@@ -214,7 +214,7 @@ li.coming-soon a::after {
 }
 
 .table-of-contents__link {
-  text-decoration: none;
+  text-decoration: underline;
```
Comment (charlesBochet) [inline docs/src/css/custom.css:217]: @nimraahmed where is this link used?

**Review by charlesBochet — APPROVED** [review]
LGTM, thanks

**Pull request merged**